### ISSUE TITLE
decreased esophagus squamous cell carcinoma incidence

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3583]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

decreased esophagus squamous cell carcinoma incidence

>2. Synonyms (e.g., absent spleen)

reduced esophagus squamous cell carcinoma incidence; decreased esophageal squamous cell carcinoma incidence; reduced esophageal squamous cell carcinoma incidence

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

less than the expected number of squamous cell carcinomas in the esophageal tissue, usually in the form of a distinct mass, in a specific population in a given time period

PMID:32123074

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

decreased alimentary system tumor incidence (MP:0012424)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by anna-anagnostop** [comment]
Add new term:
decreased esophagus squamous cell carcinoma incidence (a child of decreased alimentary system tumor incidence MP:0012424)
DEF: less than the expected number of esophageal carcinomas derived from stratified squamous epithelium, occurring in a specific population in a given time period

Also, add new term:
increased esophagus squamous cell carcinoma incidence (a child of increased alimentary system tumor incidence MP:0010292 and increased squamous cell carcinoma incidence MP:0004207)
DEF: greater than the expected number of esophageal carcinomas derived from stratified squamous epithelium, occurring in a specific population in a given time period

**Comment by anna-anagnostop** [comment]
decreased esophagus squamous cell carcinoma incidence MP:0031400
increased esophagus squamous cell carcinoma incidence MP:0031401
Added synonyms using British spelling oesophagus/oesophageal as appropriate.
